.gitignore

```diff
@@ -30,3 +30,10 @@
 ###< qossmic/deptrac-shim ###
 
 /node_modules/
```

Comment:
Deze staat er nu dubbel in met de introductie van webpack encore

Reply:
check